### ISSUE TITLE
NTR: 3-hydroxyanthranilic acid (3HAA) high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8405]
New term request 3-hydroxyanthranilic acid (3HAA) high in blood (elthat_171215055045):   UMLS: 4540006

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000703
- label: Elevated circulating 3-hydroxyanthranilic acid concentration
- parents: [Abnormal circulating proteinogenic amino acid derivative concentration (HP:0033108)](https://hpo.jax.org/app/browse/term/HP:0033108)
- pmid: PMID:22084587
- def: The concentration of 3-hydroxyanthranilic acid in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8405
- synonyms:
